extensions/cornerstone/src/components/WindowLevelActionMenu/Colormap.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { ReactElement, useCallback, useEffect, useRef, useState } from 'react';
+import { AllInOneMenu, ButtonGroup, SwitchButton } from '@ohif/ui';
+import { StackViewport } from '@cornerstonejs/core';
+import { CommandsManager, ServicesManager } from '@ohif/core';
+import { ColorMapPreset } from './WindowLevelActionMenu';
+
+export type ColormapProps = {
+  viewportId: string;
+  commandsManager: CommandsManager;
+  serviceManager: ServicesManager;
+  colormaps: Array<ColorMapPreset>;
+  displaySets: Array<any>;
+};
+
```

Comment:
put this and the 

export type ColorbarProps = {
  viewportId: string;
  commandsManager: CommandsManager;
  serviceManager: ServicesManager;
  displaySets: Array<any>;
  colorbarProperties: {
    width: string;
    colorbarTickPosition: string;
    colorbarContainerPosition: string;
    colormaps: Array<ColorMapPreset>;
    colorbarInitialColormap: string;
  };
};

inside the types folder in this extension, I think they belong there